app-emulation/playonlinux/playonlinux-4.3.4.ebuild

```diff
@@ -0,0 +1,117 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit gnome2-utils python-single-r1
```

Comment:
Use xdg or xdg-utils for icon_cache_thing please. Should allow later EAPI to be used, too.

Reply:
xdg-utils, agreed.